plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts

```diff
@@ -371,6 +371,10 @@ const { state: cartItemState } = store(
 
 				return formatPriceWithCurrency( totalLinePrice, itemCurrency );
 			},
+
+			get isAvailableInBackorder(): boolean {
```

Comment:
One small nitpick, the correct grammar would be _on_ backorder

Reply:
https://github.com/woocommerce/woocommerce/blob/d596e0344d6eb98c860daa36909680d5a75eff35/plugins/woocommerce/src/Enums/ProductStockStatus.php#L25-L30

Reply:
You're totally right. I've just changed it in the last commit. Thanks for pointing that out!